src/components/smart_objects/include/smart_objects/object_schema_item.h

```diff
@@ -67,7 +67,8 @@ struct SMember {
           const std::string& Until = "",
           const bool IsDeprecated = false,
           const bool IsRemoved = false,
-          const std::vector<SMember>& history_vector = {});
+          const std::vector<SMember>& history_vector = {},
+          const bool IsCustom = false);
```

Comment:
@YarikMamykin  I would propose to use enum instead pof bool

Reply:
@LuxoftAKutsan fixed in e5df4ef22337bafe6ed7d68ff16c98b407f40d6c